crates/bevy_input/src/gamepad.rs

```diff
@@ -147,20 +147,19 @@ impl Default for AxisSettings {
 
 impl AxisSettings {
     fn filter(&self, new_value: f32, old_value: Option<f32>) -> Option<f32> {
+        let new_value = match new_value {
```

Comment:
nit: this is barely worth mentioning at all, but I personally prefer normal if/else statements for cases like these. I think the repeated `value if value` isn't worth the newlines saved. This reads better imo:

```rust
let new_value = if new_value <= self.positive_low && new_value >= self.negative_low {
    0.0
} else if new_value >= self.positive_high {
    1.0
} else if new_value <= self.negative_high {
    -1.0
} else {
    new_value
};
```

Reply:
I've changed it to an `if` now.